prowspecs/specs.go

```diff
@@ -71,6 +71,26 @@ var knownBranches map[string]BranchSpec = map[string]BranchSpec{
 
 		skipTrivy: false,
 	},
+	"release-1.11": {
```

Comment:
suggestion: It's correct to add tests for 1.11 here - should we also remove 1.9 tests? They're only really relevant if we want to release a new version of 1.9, which to be fair we might want to do (e.g. update base images one last time before it's EOL).

What do you think? Now I've written that I think I'm leaning towards just keeping the 1.9 tests until we actually release 1.11, since along with removing 1.9 tests we can also remove the skipTrivy flag entirely!

Reply:
I'll leave them until we've released 1.11.